src/Form/Extension/AddressTypeExtension.php

```diff
@@ -64,7 +64,12 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
     {
         $builder
             ->addEventListener(FormEvents::PRE_SET_DATA, function (FormEvent $event): void {
-                $billingAddressForm = $event->getForm()->getParent()->get('billingAddress');
+                $parentForm = $event->getForm()->getParent()
+                if ($parentForm === null) {
```

Comment:
Could you change it to `null === $parentForm`?

Reply:
sure thing, i have updated the PR